json-rpc/Cargo.toml

```diff
@@ -32,11 +30,15 @@ libra-mempool = { path = "../mempool", version = "0.1.0" }
 libra-metrics = { path = "../common/metrics", version = "0.1.0" }
 libra-proptest-helpers = { path = "../common/proptest-helpers", optional = true }
 libra-types = { path = "../types", version = "0.1.0" }
-libra-temppath = { path = "../common/temppath", version = "0.1.0", optional = true }
 libra-workspace-hack = { path = "../common/workspace-hack", version = "0.1.0" }
 proptest = { version = "0.9.6", optional = true }
 storage-interface = { path = "../storage/storage-interface", version = "0.1.0" }
 
+
+#[cfg(feature = "fuzzing")]
```

Comment:
This line does nothing, it's just a comment in a toml file.

You want to add `optional = true` to the reqwest dependency as well, and then add `"reqwest"` to the list of features enabled by fuzzing in the list on line 46 under `[features]` section.